tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -103,6 +104,13 @@ install_kernel() {
 	DESTDIR="${destdir}" PREFIX="${prefix}" "${kernel_builder}" -f -v "${kernel_version}"
 }
 
+#Install kernel asset
```

Comment:
maybe the comment here could specify that this is dragonball experimental guest kernel to separate from the comment from `install_kernel()`

Reply:
Thanks, updated!